tsfile/pom.xml

```diff
@@ -100,6 +100,12 @@
             <artifactId>iotdb-antlr</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Why this dependency is necessary in tsfile module?

Reply:
Updated